docker-compose.yaml

```diff
@@ -0,0 +1,39 @@
+#https://docs.docker.com/registry/deploying/#deploy-your-registry-using-a-compose-file
+services:
+  nginx:
+    image: "nginx:alpine"
+    ports:
+      - 5043:443
+    restart: unless-stopped
+    links:
+      - registry:registry
+    volumes:
+      - ./auth:/etc/nginx/conf.d
+      - ./auth/nginx.conf:/etc/nginx/nginx.conf:ro
+  
+  # SSL certificate management for nginx
+  certbot:
+    image: certbot/certbot
+    container_name: 'certbot'
+    volumes:
+      - ../../certbot/conf:/etc/letsencrypt
+      - ../../certbot/www:/var/www/certbot
+    restart: always
+    # This will check if your certificate is up for renewal every 12 hours as recommended by Let’s Encrypt
+    entrypoint: "/bin/sh -c 'trap exit TERM; while :; do certbot renew; sleep 12h & wait $${!}; done;'"
```

Comment:
This reminds me that we should restart nginx too, to pick up new certs. See https://github.com/Seneca-CDOT/telescope/blob/master/docker/production.yml#L139 for an example.